mkdocs.yml

```diff
@@ -0,0 +1,83 @@
+# Site
+site_name: 'Mobius'
+site_description: 'A functional reactive framework for managing state evolution and side-effects.'
+site_author: 'Spotify'
+site_url: 'https://github.com/spotify/mobius'
+
+# Repository
+repo_name: 'spotify/mobius'
+repo_url: 'https://github.com/spotify/mobius'
+edit_uri: 'tree/main/docs/'
+remote_branch: gh-pages
+
+# Navigation
+nav:
+  - 'Overview': index.md
+  - 'Concepts': concepts.md
+  - 'Objectives': objectives.md
+  - 'Mobius Workflow': the-mobius-workflow.md
+  - 'FAQ': faq.md
+  - 'Getting Started':
+      - 'Creating a loop': getting-started/creating-a-loop.md
+      - 'Mobius an RxJava': getting-started/mobius-and-rx-java.md
+      - 'Mobius and Android': getting-started/mobius-and-android.md
+      - 'Configuring a MobiusLoop': getting-started/configuring-a-mobius-loop.md
+      - 'Logging and Error Handling': getting-started/logging-and-error-handling.md
+  - 'Reference Guide':
+      - 'Mobius Loop': reference-guide/mobius-loop.md
+      - 'Model': reference-guide/model.md
+      - 'Event': reference-guide/event.md
+      - 'Effect': reference-guide/effect.md
+      - 'Update': reference-guide/update.md
+      - 'Effect Handler': reference-guide/effect-handler.md
+      - 'Init': reference-guide/init.md
+      - 'Event Source': reference-guide/event-source.md
+  - 'Patterns':
+      - 'Immutability': patterns/immutability.md
+      - 'Pure vs Impure Functions': patterns/pure-vs-impure-functions.md
+      - 'Models': patterns/models.md
+      - 'Events and Effects': patterns/events-and-effects.md
+  - 'Prior Art': prior-art.md
+  - 'License': license.md
+
+# Theme
+theme:
+  name: 'material'
+  language: 'en'
+  icon:
+    logo: fontawesome/brands/spotify
+    repo: fontawesome/brands/github
```

Comment:
You can change the logos ofc. I think I got it right though 😊

The `logo` is what is shown on the left and the `repo` on the right.

![Screen Shot 2022-03-14 at 1 12 28 PM](https://user-images.githubusercontent.com/4332753/158275672-38118800-65c9-4cb2-83a4-52f5431c3f60.png)

I did not set the favicon so the default MkDocs material favicon is used. You can change that if you want.